activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelingRestExceptionHandler.java

```diff
@@ -82,6 +83,13 @@ public void handleModelNameConflictException(ModelNameConflictException ex, Http
         response.sendError(CONFLICT.value(), ex.getMessage());
     }
 
+    @ExceptionHandler({ ModelNameInvalidException.class })
+    public void handleModelNameInvalidException(ModelNameInvalidException ex, HttpServletResponse response)
+        throws IOException {
```

Comment:
Is the `throws IOException` needed here?

Reply:
Yes because response.sendError throws IOException